fsharp-backend/src/LibBackend/Init.fs

```diff
@@ -43,7 +43,7 @@ let waitForDB () : Task<unit> =
         success <- true
       with
       | _ -> do! Task.Delay 1000
-      return ()
+    return ()
```

Comment:
ha!

Reply:
`return` actually exits the loop here, right?

Reply:
I'm not sure what to expect, honestly, though this seems like a worthwhile fiddle.